source/dub/commandline.d

```diff
@@ -1236,7 +1241,29 @@ class RemoveCommand : FetchRemoveCommand {
 		auto package_id = free_args[0];
 		auto location = dub.defaultPlacementLocation;
 
-		dub.remove(package_id, m_version, location, m_forceRemove);
+		size_t resolveVersion(in Package[] packages) {
+			writeln("Select version of '", package_id, "' to remove from location '", location, "':");
+			foreach(i, pack; packages)
+				writeln(i, ". ", pack.version_);
```

Comment:
Me too, but 95% of the time I do dlang PR reviews on my phone, which highly unusable for dev work :/